components/button/index.css

```diff
@@ -130,7 +130,7 @@ governing permissions and limitations under the License.
   }
 
   &.spectrum-Button--iconOnly {
-    min-inline-size: unset;
+    min-inline-size: var(--mod-button-height, var(--spectrum-button-height));
     padding: calc(var(--mod-button-edge-to-visual-only, var(--spectrum-button-edge-to-visual-only)) - var(--mod-button-border-width, var(--spectrum-button-border-width)));
     border-radius: 50%;
```

Comment:
Noticed a 2 pixel difference in height with the icon only buttons and the buttons with a label. 

I believe it is coming from 
`padding-block-end: calc(var(--mod-button-padding-label-bottom, var(--spectrum-button-padding-label-bottom)) - var(--mod-button-border-width, var(--spectrum-button-border-width)));` 
on 
`.spectrum-Button-label`

Reply:
We'll look to fix this in a separate ticket. Thanks for catching it!